projects/storefrontlib/src/cms-components/myaccount/update-password/components/update-password-form/update-password-form.component.ts

```diff
@@ -1,31 +1,23 @@
 import { Component, EventEmitter, OnInit, Output } from '@angular/core';
-import {
-  AbstractControl,
-  FormBuilder,
-  FormGroup,
-  ValidationErrors,
-  Validators,
-} from '@angular/forms';
-import { FormUtils } from '../../../../../shared/utils/forms/form-utils';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
 import { CustomFormValidators } from '../../../../../shared/utils/validators/custom-form-validators';
 @Component({
   selector: 'cx-update-password-form',
   templateUrl: './update-password-form.component.html',
 })
 export class UpdatePasswordFormComponent implements OnInit {
```

Comment:
Breaking changes to document:
- changed output submited to submitted 
- removed isNotValid and isPasswordConfirmNotValid

Reply:
Done